### DISCUSSION TITLE
Why do you repackage dependencies into clickhouse-kafka-connect

### DISCUSSION THREAD

**Discussion by kpavlov** [ClickHouse/clickhouse-kafka-connect#89] [Q&A]
Clickhouse-kafka-connect.jar 0.0.10-beta contains classes from reactive-streams, project reactor and other dependencies.
Is there any specific reason and can you stop doing this?

**Comment by Paultagoras** [comment]
We repackage them because Kafka-Connect doesn't include the required dependencies.
